### PR TITLE
Dialog box for deletion confirmation

### PR DESCRIPTION
**Pull request by chirag-jn**
Dialog box added to confirm before deletion in offline files to prevent any deletion by mistake.

### CODE REVIEW

**Comment by chirag-jn** [comment]
The app is successfully running after the changes made. Maybe there's some other issue. I'll look into it.

File: src/main/java/org/amahi/anywhere/activity/OfflineFilesActivity.java [hunk 1]
```diff
@@ -180,7 +182,23 @@ private View getParentView() {
 
     @Subscribe
     public void onFileDeleting(OfflineFileDeleteEvent event) {
-        deleteFileFromOfflineStorage(event.getFile());
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setTitle("Delete file");
```
Comment (octacode) [inline src/main/java/org/amahi/anywhere/activity/OfflineFilesActivity.java]: I would recommend putting these strings visible to the user in strings.xml

Comment (chirag-jn) [reply src/main/java/org/amahi/anywhere/activity/OfflineFilesActivity.java]: Okay, thanks. I will do that.

**Comment by codedentwickler** [comment]
@octacode @cpg I think this PR can be merged now, all the proposed changes has been completed. We have too many open PRs.

**Comment by chirag-jn** [comment]
@cpg I guess we can merge this Pr now. Its review is completed a long time back.

**Pull request merged**